runtime/mainnet/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = { workspace = true }
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-wasm-builder = { workspace = true, optional = true }
+substrate-wasm-builder = { workspace = true, optional = true, features = ["metadata-hash"] }
```

Comment:
@valentinfernandez1 @metricaez please, take a look at [this](https://github.com/paritytech/polkadot-sdk-parachain-template/blob/master/runtime/Cargo.toml#L187-L198).

I'd like your input on whether it is a good idea to include the metadata hash in non-production builds, since it significantly increases build time. I don't mind to enable a flag in the deterministic runtime CI step only, and leave our local builds and other non-related CI-tasks free.